sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+friendly_name: Newtab Daily Aggregations
+description: |-
+  A daily aggregation of newtab metrics.
+  The granularity is one row per day per each of the following dimensions:
+      submission_date
+      app_version
+      os
+      channel
+      locale
+      browser_version
+      country
+      newtab_homepage_category
+      newtab_newtab_category
+      organic_content_enabled
+      sponsored_content_enabled
+      sponsored_topsites_enabled
+      organic_topsites_enabled
+      newtab_search_enabled
+owners:
+  - gkatre@mozilla.com
+labels:
+  application: newtab
+  incremental: true
+  schedule: daily
+  dag: bqetl_newtab
+  owner1: gkatre
+  table_type: aggregate
+scheduling:
+  dag_name: bqetl_newtab
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
```

Comment:
This table is small enough that it'd be nice if we didn't require the filter, it won't be too expensive to query the whole thing